lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -82,7 +82,18 @@ def call(env)
 
           # call the rest of the stack
           status, headers, response = @app.call(env)
-          [status, headers, response]
+
+          wrapped_response = response
+
+          if defined?(::Rack::BodyProxy)
```

Comment:
Can you explain the significance of this check? Is it just a compatibility check, or does it represent something else?

Reply:
yes, just a compatibility check